lib/galaxy/__init__.py

```diff
@@ -91,5 +91,3 @@ class NullHandler( logging.Handler ):
         def emit( self, record ):
             pass
     logging.NullHandler = NullHandler
-
-import galaxy.eggs
```

Comment:
I'd like to avoid #noqa whenever possible, since it'll hide other errors we will want to see.  I will look into what we should do here and get back to you.